package.json

```diff
@@ -28,42 +31,49 @@
   ],
   "dependencies": {},
   "peerDependencies": {
-    "react": "0.13.x || 0.14.x || 15.x.x",
-    "firebase": "2.x.x || 3.x.x"
+    "react": "15.x.x",
+    "firebase": "3.x.x"
   },
   "devDependencies": {
-    "babel-cli": "^6.14.0",
-    "babel-core": "^6.14.0",
-    "babel-preset-es2015": "^6.14.0",
-    "babel-preset-react": "^6.11.1",
-    "babel-register": "^6.18.0",
-    "chai": "^3.0.0",
-    "chai-as-promised": "^6.0.0",
-    "coveralls": "^2.11.2",
-    "eslint": "^3.10.2",
-    "eslint-config-airbnb": "^13.0.0",
+    "babel-cli": "^6.23.0",
+    "babel-core": "^6.23.0",
+    "babel-jest": "^18.0.0",
+    "babel-preset-babili": "0.0.11",
+    "babel-preset-es2015": "^6.22.0",
+    "babel-preset-react": "^6.23.0",
+    "babel-register": "^6.23.0",
+    "coveralls": "^2.11.16",
+    "eslint": "^3.16.0",
+    "eslint-config-airbnb": "^14.0.0",
     "eslint-plugin-import": "^2.2.0",
-    "eslint-plugin-jsx-a11y": "^2.2.3",
-    "eslint-plugin-react": "^6.7.1",
+    "eslint-plugin-jsx-a11y": "^4.0.0",
+    "eslint-plugin-react": "^6.10.0",
     "firebase": "3.x.x",
     "gulp": "^3.9.0",
     "gulp-babel": "^6.1.2",
     "gulp-eslint": "^3.0.1",
     "gulp-exit": "0.0.2",
     "gulp-ext-replace": "^0.3.0",
     "gulp-istanbul": "^1.1.1",
-    "gulp-mocha": "^3.0.1",
     "gulp-uglify": "^2.0.0",
-    "jsdom": "3.x.x",
+    "jest": "^18.1.0",
+    "npm-watch": "^0.1.8",
     "react": "15.x.x",
     "react-addons-test-utils": "15.x.x",
     "react-dom": "15.x.x",
-    "run-sequence": "^1.1.1",
-    "sinon": "^1.15.4",
-    "sinon-chai": "^2.8.0"
+    "react-test-renderer": "^15.4.1",
+    "run-sequence": "^1.1.1"
+  },
+  "watch": {
+    "test": "{src,tests}/**/*.js"
   },
   "scripts": {
-    "test": "gulp test",
-    "travis": "gulp"
+    "test": "npm run lint ; npm run build ; jest --coverage",
+    "lint": "eslint src/ tests/",
```

Comment:
perhaps we can use the `node_modules/.bin/` binaries here so that they do not need global installation